README.md

```diff
@@ -27,4 +27,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
\ No newline at end of file
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind welcome!
+
+## Github markdown resources
```

Comment:
Neatly written and done! Well done 👍
But your new section is in the wrong place. Check the issue again to see where you should have added it.

Reply:
I have fixed it,

Reply:
💯🙂👍